pkg/trait/service.go

```diff
@@ -79,7 +79,7 @@ func (t *serviceTrait) Apply(e *Environment) (err error) {
 	}
 	port := corev1.ServicePort{
 		Name:       httpPortName,
-		Port:       80,
+		Port:       int32(t.Port),
```

Comment:
@astefanutti I had to change as otherwise  the port is set on the trait is not taken into account but I do not know the reason of the change, can you please validate ?

Reply:
@lburgazzoli I think that it should work without this change. I think the original issue is due to the host bound to `localhost`. I've tested the sample with service port 80 and it works on my local setup (Minishift). Some Linux capabilities exist for port binding on pods but I do not know of any restrictions applicable to services.

Reply:
reverted